Homework/homework1/task2.py

```diff
@@ -0,0 +1,22 @@
+"""
+Given a cell with "it's a fib sequence" from slideshow,
+
+    please write function "check_fib", which accepts a Sequence of integers, and
+    returns if the given sequence is a Fibonacci sequence
+We guarantee, that the given sequence contain >= 0 integers inside.
+"""
+from typing import Sequence
+
+
+def _check_window(x: int, y: int, z: int) -> int:
+    return (x + y) == z
+
+
+def check_fibonacci(data: Sequence[int]) -> bool:
+    if len(data) < 3 or data[1] == 0:
```

Comment:
Lets think which part of the algo we need to cover by tests.

- `len < 3`  values before border.  So we need to check (border - 1).  Other possible values looks special so they need to be tested too. Both true and false scenarious should be considered.
- len >=3 this is a border, need to check it, border + 1 and some average values. Both possible and negative scenarious should be considered.

After you will get all this scenarious, you will need to copy almost all false scenarious and pot an 0 to the second item. 

, len > 3 and

Reply:
Okey, got it, thank you! 
About conflict that must be resolved. It happened after moving .flake8 to the root. As I understood I will need to merge this branch via command line because of this conflict. Right?

Reply:
git checkout check1
git merge master
git push check1
merge PR